src/main/java/com/sfuentes/Stationery.java

```diff
@@ -0,0 +1,50 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Stationery {
```

Comment:
I'm not sure about the name you chose for this, stationery translates to "articulos de papeleria", maybe if you add shopt it would be more clear: StationeryShop
or something different like PrintKiosk, but I'm not 100% sure though.

Reply:
fixed.